### PR TITLE
Making spec/actions lint free

### PR DESCRIPTION
**Pull request by kepta**
To prepare for modularization of iD, I believe it would be good if we could lint all our test folder.

### CODE REVIEW

File: test/spec/actions/merge_remote_changes.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
-describe("iD.actions.MergeRemoteChanges", function () {
+/* global locale: true */
```
Comment (kepta) [inline test/spec/actions/merge_remote_changes.js]: @bhousel I would want you to take a closer look at this diff.

File: test/spec/actions/merge_remote_changes.js [hunk 2]
```diff
@@ -367,27 +369,27 @@ describe("iD.actions.MergeRemoteChanges", function () {
...
                     remoteTags = {foo: 'foo', bar: 'bar_remote'},  // same foo, added bar
                     remoteLoc = [2, 2],                            // moved node
                     local = base.entity('a').update({tags: localTags}),
                     remote = base.entity('a').update({tags: remoteTags, loc: remoteLoc, version: '2'}),
                     localGraph = makeGraph([local]),
                     remoteGraph = makeGraph([remote]),
-                    action = iD.actions.MergeRemoteChanges('a', localGraph, remoteGraph),
-                    result = action(localGraph);
+                    action = iD.actions.MergeRemoteChanges('a', localGraph, remoteGraph);
+                    action(localGraph);
```
Comment (kepta) [inline test/spec/actions/merge_remote_changes.js]: @bhousel remote was unused var here, so I removed it.

File: test/spec/actions/merge_remote_changes.js (hunk 1 above)
Comment (bhousel) [inline test/spec/actions/merge_remote_changes.js]: Yes, I think that's right..

**Comment by bhousel** [comment]
This is awesome!  I've wanted to do it for a while..

Can you also make sure the spec files get linted [when `npm run test` is called](https://github.com/openstreetmap/iD/blob/master/package.json#L12)?

**Comment by kepta** [comment]
@bhousel sure.

**Comment by kepta** [comment]
@bhousel updated the PR.

**Comment by bhousel** [comment]
npm run commands look great, merge it if you are happy with it.. :shipit:

**Pull request merged**